bob-backend/src/pearl/group.rs

```diff
@@ -498,6 +498,36 @@ impl Group {
         result
     }
 
+    pub(crate) async fn corrupted_blobs_count(&self) -> usize {
+        let mut corrupted_blobs = 0;
+        let holders_guard = self.holders();
```

Comment:
This variable is not needed. You can get `holders` in one line: 
```
let holders = self.holders().read().await;
```

Reply:
It will cause "temporary value dropped while borrowed" error, cause self.holders() value is dropped at the end of the statement

Reply:
Ok. Thank you for explanation!